test/config.js

```diff
@@ -671,7 +675,13 @@ test('disableInstrumentations', function (t) {
       }
 
       for (const mod of modules) {
-        require(mod)
+        if (Array.isArray(mod)) {
```

Comment:
Because the modules array was flattened above, this shouldn't be necessary right?

Reply:
Fixed